apps/io-func-wallet-solution-backend/openapi.yaml

```
@@ -170,12 +173,15 @@ components:
       properties:
         challenge:
           type: string
+          minLength: 1
           format: binary
         key_attestation:
           type: string
+          minLength: 1
           format: binary
         hardware_key_tag:
           type: string
+          minLength: 1
```

I propose to eliminate the format since it could create some problems with the generators

```suggestion
          type: string
          minLength: 1
        key_attestation:
          type: string
          minLength: 1
        hardware_key_tag:
          type: string
          minLength: 1
```

---

the generators seem to be working. Do we want to give it a try and leave it as it is?

---

ok

---

Ok, we can leave it like this then. Let's try.